src/main/java/randoop/main/GenInputsAbstract.java

```diff
@@ -517,6 +517,14 @@ public Limits(int time_limit, int attempted_limit, int generated_limit, int outp
     ALL
   }
 
+  /**
+   * Enable the "bloodhound" technique from the GRT paper, which prioritizes methods with lower
+   * branch coverage.
+   */
+  @Unpublicized
+  @Option("Enable bloodhound to prioritize methods with lower branch coverage.")
+  public static boolean enable_bloodhound = true;
```

Comment:
This should be changed to `false` before merging -- unless we have completed experiments showing that Bloodhound is always superior, but I suspect we will merge before completing such experiments.